.github/workflows/android.yml

```diff
@@ -29,9 +31,9 @@ jobs:
         id: vars
         shell: bash
         run: |
-          echo "ANDROID_NDK_VERSION=22.1.7171670" >> $GITHUB_ENV
-          echo "ANDROID_BUILD_TOOLS_VERSION=29.0.2" >> $GITHUB_ENV
-          echo "ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk/22.1.7171670" >> $GITHUB_ENV
+          echo "ANDROID_NDK_VERSION=23.1.7779620" >> $GITHUB_ENV
```

Comment:
This is needed to avoid this error when running the app:
```
--------- beginning of crash
04-29 10:31:43.274 28217 28246 E AndroidRuntime: FATAL EXCEPTION: qtMainLoopThread
04-29 10:31:43.274 28217 28246 E AndroidRuntime: Process: org.qtproject.example.qfield, PID: 28217
04-29 10:31:43.274 28217 28246 E AndroidRuntime: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "__emutls_get_address" referenced by "/data/app/~~KGkD6tWZgcI6b71aRktJJw==/org.qtproject.example.qfield-dUEJuMCJyq3L6E3yOm1XLw==/lib/arm64/libQt6Core_arm64-v8a.so"...
```